pydocx/export/base.py

```diff
@@ -367,11 +464,59 @@ def export_sdt_content_block(self, sdt_content_block):
         return self.export_sdt_content(sdt_content_block)
 
     def export_tab_char(self, tab_char):
-        return
-        yield
+        pass
 
     def export_numbering_span(self, numbering_span):
         return self.yield_nested(numbering_span.children, self.export_node)
 
     def export_numbering_item(self, numbering_item):
         return self.yield_nested(numbering_item.children, self.export_node)
+
+    def parse_simple_field_instr(self, instr):
+        # TODO this might be better implemented directly in the SimpleField class
+        if not instr.strip():
+            return
+        # Matches: COMMAND args ...
+        m = re.match('^\s*([^\s]+)\s+(.*)$', instr)
+        if not m:
+            return
+        field_type = m.group(1)
+        raw_field_args = m.group(2)
+        if not raw_field_args:
+            return field_type, None
+        m = re.findall(
+            # Matches: One two "foo bar" baz
+            r'(?:\s?\s*(?:"([^"]+)"|([^\s]+))+)',
```

Comment:
No, It's my own. The outer unnamed group is actually unneeded, and it can be simplified further: `\s*(?:"([^"]+)"|([^\s]+))+`

Matches all sequences that:

1. starts with 0 or more white space, followed by
2. a sequence that starts with a " and ends with a ", or
3. a sequence of one or more non-space characters

I think I want to pull this regex out into a function and put some tests on it